src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -22,8 +22,8 @@ private void TestWindowApisUnix()
             // This tests the hardcoded implementations of these APIs on Unix.
             using (Process p = Process.GetCurrentProcess())
             {
-                Assert.True(p.Responding);
-                Assert.Equal(string.Empty, p.MainWindowTitle);
+                Assert.Throws<PlatformNotSupportedException>(() => p.Responding);
```

Comment:
@krwq if we were writing .NET v1 for Unix and Windows together we might make it throw for Unix however the context is that we have 15 years of libraries already written for Windows and we want to give them a chance to magically work on Unix. That means we can't afford to be quite as "pure" in many cases...